preview_generator/preview/builder/office__libreoffice.py

```diff
@@ -147,6 +148,7 @@ def convert_office_document_to_pdf(
                         ],
                         stdout=DEVNULL,
                         stderr=STDOUT,
+                        start_new_session=True,
```

Comment:
What does this do?

Reply:
"If start_new_session is true the setsid() system call will be made in the child process prior to the execution of the subprocess. (POSIX only)"

https://docs.python.org/3/library/subprocess.html 

I don't fully understood the subprocess thing, but without this, there is no "group/session id" related to the process we do want to close, so killing the group doesn't work.